app/assets/stylesheets/shared/form.scss

```diff
@@ -235,7 +235,17 @@
 }
 
 .ss-main.form-select.is-invalid:not([multiple]):not([size]) {
+  padding-right: $form-select-feedback-icon-padding-end;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill='none' stroke='%23323233' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='m2 5 6 6 6-6'/%3e%3c/svg%3e"),
```

Comment:
I tried leveraging the Sass-variable found here: https://getbootstrap.com/docs/5.3/forms/validation/#sass-variables `$form-feedback-icon-invalid` but when I used it, the images all went blank

But the image never appeared.